pkg/assets/rte/assets.go

```diff
@@ -2,16 +2,33 @@ package rte
 
 import (
 	_ "embed"
+
+	"github.com/k8stopologyawareschedwg/deployer/pkg/deployer/platform"
+)
+
+const (
+	OCPVersion411 = "4.11"
 )
 
 //go:embed selinuxinstall.service.template
 var SELinuxInstallSystemdServiceTemplate []byte
 
-//go:embed selinuxpolicy.cil
-var SELinuxPolicy []byte
+//go:embed selinuxpolicy-ocp410.cil
+var SELinuxPolicyOCP410 []byte
+
+//go:embed selinuxpolicy-ocp411.cil
+var SELinuxPolicyOCP411 []byte
 
 //go:embed hookconfigrtenotifier.json.template
 var HookConfigRTENotifier []byte
 
 //go:embed rte-notifier.sh
 var NotifierScript []byte
+
+func GetSELinuxPolicy(ver platform.Version) []byte {
```

Comment:
if I'm right (see below), it seems this must return `([]byte, error)`

Reply:
I am not sure I understand, we are returning either `SELinuxPolicyOCP411` or `SELinuxPolicyOCP412`  which are both of type []byte. Why you we need to return `([]byte, error)` ?

Reply:
because this function should have failed when fed with `plat` casted as `platform.Version`, and we swallow the error, so the test didn't fail, which is bad

Reply:
Fixing it now.

Reply:
yes, it's on me, sorry for misleading you. I was too optimist!